skeleton-navigation/src/my-app.less__if_less

```diff
@@ -0,0 +1,48 @@
+body {
+  font-family: 'Source Sans Pro', sans-serif;
+}
+
+nav {
+  background: linear-gradient(135deg, #6d50a2 0%, #ed2b88 100%);
+  display: flex;
+}
+
+a {
+  padding: 10px;
+  text-decoration: none;
+  color: black;
+}
+a:hover {
```

Comment:
Less file can be cleaned up to use nested class. There are few others can be merged.

Reply:
done